build-tools/package.json

```diff
@@ -21,6 +21,7 @@
 		"changelog": "conventional-changelog -p conventionalcommits -i CHANGELOG.md -s -r 2 --commit-path . -t build-tools_v --context lerna.json",
 		"ci:build": "lerna run build:compile --stream",
 		"ci:build:docs": "lerna run ci:build:docs --stream --parallel",
+		"ci:lint": "lerna run eslint --no-sort --stream",
```

Comment:
Should this be `ci:eslint` or something instead, since it doesn't run a complete lint?

Reply:
You're right, this should run the lint task, not eslint. WIll fix.

Reply:
No, this was correct as it was. We don't want to run the lint task in CI. We want to run eslint, which does not run prettier AND lint. We're cheating because we know that eslint is the only command that the lint task runs (besides prettier). But the alternative is making _every_ package do the ci/local split.

Reply:
Sorry, my feedback was ambiguous. I was just suggesting a name change. Since this only runs eslint, I was suggesting naming the scripe "ci:eslint" instead of "ci:lint".

Reply:
Ahhhh, my mistake! Thanks for clarifying.